gmail/1.1/modules/ROOT/pages/gmail-connector-reference.adoc

```diff
@@ -5524,6 +5527,21 @@ Configures the repeatable file-store streaming strategy by which Mule keeps a po
 ** GB | Unit for the *In Memory Size* field. |  |
 |===
 
+[[EmailAttachment]]
```

Comment:
Same about this part, we should remove it if we remove the Send email operation.